integrations/pagerduty/clients/pagerduty.py

```diff
@@ -215,3 +216,39 @@ async def get_incident_analytics(self, incident_id: str) -> dict[str, Any]:
         except httpx.HTTPError as e:
             logger.error(f"HTTP occurred while fetching incident analytics data: {e}")
             return {}
+
+    async def get_service_analytics(self, service_id: str) -> dict[str, Any]:
+        logger.info(f"Fetching analytics for service: {service_id}")
+        url = f"{self.api_url}/analytics/metrics/incidents/services"
+
+        date_ranges = get_date_range_for_last_n_months(3)
```

Comment:
why 3? is this a user request / product decision? we might want to have it configurable 🤔